src/server.tsx

```diff
@@ -327,21 +260,7 @@ app.get(
     '/epic.js',
     async (req: express.Request, res: express.Response, next: express.NextFunction) => {
         try {
-            const path = isDev ? '/../dist/modules/Epic.js' : '/modules/Epic.js';
-            const module = await fs.promises.readFile(__dirname + path);
-            res.type('js');
-            res.send(module);
-        } catch (error) {
-            next(error);
-        }
-    },
-);
-
-app.get(
-    '/banner.js',
```

Comment:
we should check which endpoint subscriptions and contributions team are planning to use for their banners (not including the Aus moment)

Reply:
Okay will do. In general, I'm more for not adding code until it is actually in use, but perhaps it is closer than I realised in this case!

Reply:
For the banner component module we'll have separate endpoints for contributions and subscriptions (coming soon!)
It's fine to remove this one for now